redash/handlers/query_results.py

```diff
@@ -131,6 +131,13 @@ def post(self):
 ONE_YEAR = 60 * 60 * 24 * 365.25
 
 
+class QueryResultDropdownResource(BaseResource):
+    def get(self, query_id):
+        data = dropdown_values(query_id, self.current_org)
+        headers = {'Content-Type': "application/json"}
+        return make_response(json_dumps(data), 200, headers)
```

Comment:
`BaseResource` takes care of converting an object to JSON. So you can do just:

```python
    return dropdown_values(...)
```

Reply:
95b56109